subsys/testsuite/Kconfig

```diff
@@ -144,4 +144,22 @@ config TEST_ARM_CORTEX_M
 	  Note that by default, when building with ARM_SECURE_FIRMWARE
 	  set, these exceptions are set to target the Non-Secure state.
 
+config TEST_NEGATIVE_FATAL_HANDLER
```

Comment:
Does it make sense to use NEGATIVE_TEST_FATAL_HANDLER, rather than TEST_NEGATIVE_FATAL_HANDLER?

Reply:
Yes, I think so. Updated it already.